README.md

```diff
@@ -25,3 +25,39 @@ sudo apt install build-essential
 ```
 go build -o rpc ./cmd
 ```
+
+### Docker image
+
+```bash
+docker build -t openamt:rpc-go .
+```
+
+and then you can run it using
+
+```bash
+$ docker run --rm -it --device /dev/mei0 openamt:rpc-go amtinfo
```

Comment:
mmm, I wonder if i should also make it `--net host`

Reply:
I think its ok without